app/models/invoice.rb

```diff
@@ -465,10 +464,10 @@ def charge!(automatic = true)
       # REFACTOR: Move the logic to InvoiceMessenger
       if charging_retries_count < MAX_CHARGE_RETRIES
         if unpaid?
-          logger.info("Retrying #{self.id}, unpaid")
+          logger.info("Invoice #{id} is unpaid, will be retried")
           save!
         else
-          logger.info("Retrying #{self.id}, marking as unpaid")
+          logger.info("Invoice #{id} is marked as unpaid, will be retried")
```

Comment:
Just to make clear this is something that we do now vs something that was already the case and continues to be the case. Previous messages were unclear to me.

```suggestion
          logger.info("Marking invoice #{id} as unpaid, will be retried")
```

Reply:
Yep, sure. What I really didn't like is this "Retrying" thing, it was just misleading, as we are not retrying anything at this point.
Your suggestions seems reasonable to me.